aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,179 @@
+            }
+        })
+        try:
+            base_connection_id = base_connection_res["id"]
+            print("baseConnectionId " + base_connection_id + "has been created")
+        except KeyError:
+            raise RuntimeError(f"Error when creating base connection: {base_connection_res}")
+
+        target_res = self.flow_conn.createTargetConnection(
+            data={
+                "name": f"[CMLE][Week2] Data Landing Zone target connection created by {self.username}",
+                "baseConnectionId": base_connection_id,
+                "params": {
+                    "mode": "Server-to-server",
+                    "compression": compression_type,
+                    "datasetFileType": data_format,
+                    "path": export_path
+                },
+                "connectionSpec": {
+                    "id": connection_spec_id,
+                    "version": "1.0"
+                }
+            }
+        )
+        try:
+            target_connection_id = target_res["id"]
+            print("targetconnectionId " + target_connection_id + " has been created")
+        except KeyError:
+            raise RuntimeError(f"Error when creating target connection: {target_res}")
+
+        if on_schedule:
+            schedule_params = {
+                "interval": 3,
+                "timeUnit": "hour",
+                "startTime": int(time.time())
+            }
+        else:
+            schedule_params = {
+                "interval": 1,
+                "timeUnit": "day",
+                "startTime": int(time.time() + 60*60*24*365)
+            }
+        flow_spec_id = "cd2fc47e-e838-4f38-a581-8fff2f99b63a"
+        flow_obj = {
+            "name": f"[CMLE][Week2] Flow for Featurized Dataset to DLZ created by {self.username}",
+            "flowSpec": {
+                "id": flow_spec_id,
+                "version": "1.0"
+            },
+            "sourceConnectionIds": [
+                source_connection_id
+            ],
+            "targetConnectionIds": [
+                target_connection_id
+            ],
+            "transformations": [],
+            "scheduleParams": schedule_params
+        }
+        flow_res = self.flow_conn.createFlow(
+            obj = flow_obj,
+            flow_spec_id = flow_spec_id
+        )
+        try:
+            dataflow_id = flow_res["id"]
+            print("dataflowId " + dataflow_id + " has been created")
+        except KeyError:
+            raise RuntimeError(f"Error when creating data flow: {flow_res}")
+
+        #save dataflow_id to config
+        self.utils.save_field_in_config("Platform", "dataflow_id", dataflow_id)
+        return dataflow_id
+
+    def createFlowRun(self, dataflow_id, dataset_id, on_schedule):
+        if not on_schedule:
+            res = self.retryOnNotReadyException(dataflow_id = dataflow_id, dataset_id= dataset_id)
+            flowRunId = res["destinations"][0]["datasets"][0]["statusURL"].rsplit('/', 1)[-1]
```

Comment:
Let's keep a consistent naming convention for variable, in the rest of the code you use `foo_bar_baz` but here you use `fooBarBaz`. Usually the Python convention is to use `foo_bar_baz`.